.reaction/jsdoc/templates/static/README.md

```diff
@@ -0,0 +1,32 @@
+# Reaction Commerce API Docs
+
+## API Docs
+
+Start here:
+
+- [Reaction Components API](Components.html)
+- [Reaction Core schemas](schemas.html)
+
+> **ProTip**: You can generate docs locally and work offline. Run `npm run docs` and open `file:///tmp/reaction-docs/index.html` in any browser.
```

Comment:
@machikoyasuda 
Not a blocker, but have we confirmed that this file reference is correct for windows users?

Reply:
@zenweasel Can you test this in Windows for me?